inc/saibuffer.h

```diff
@@ -328,14 +328,23 @@ typedef enum _sai_buffer_profile_attr_t
      * default to 0. */
     SAI_BUFFER_PROFILE_ATTR_XOFF_TH,
 
-    /** set the buffer profile XON threshold in byte [sai_uint32_t]
+    /** set the buffer profile XON non-hysteresis threshold in byte [sai_uint32_t]
      * Valid only for ingress PG (CREATE_AND_SET)
-     * Generate XON when the total buffer usage of this PG
-     * is less this threshold and available buffer in the PG buffer
-     * is larger than the XOFF threahold.
+     * Ensures to generate XON, and ignore hysteresis, when the total 
+     * buffer usage of this PG is less than this threshold and available 
+     * buffer in the PG buffer is larger than the XOFF threshold.
      * default to 0. */
     SAI_BUFFER_PROFILE_ATTR_XON_TH,
 
+    /** set the buffer profile XON hysteresis threshold in byte [sai_uint32_t]
```

Comment:
Please could you give an example for this attribute. I think already there is a hysteresis in picture due to use of XOFF and XON.